common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -996,69 +989,142 @@ div.problem {
     border: 1px solid $gray-l3;
   }
 
+  .message {
+    font-size: inherit;
+  }
+
   .detailed-solution {
-    > p:first-child {
-      @extend %t-strong;
-      color: $gray;
-      text-transform: uppercase;
-      font-style: normal;
-      font-size: 0.9em;
-    }
+    > p {
```

Comment:
I would state this as `.dedicated-solution > p {` and remove this level of nesting, as `dedicated-solution` is given no rules of its own - I think it's cleaner and gives context to what `>` means